website/docs/r/registry_module.html.markdown

```diff
@@ -36,6 +36,35 @@ resource "tfe_registry_module" "test-registry-module" {
 }
 ```
 
+Create private registry module with tests enabled:
+
+```hcl
+resource "tfe_organization" "test-organization" {
+  name  = "my-org-name"
+  email = "admin@company.com"
+}
+
+resource "tfe_oauth_client" "test-oauth-client" {
+  organization     = tfe_organization.test-organization.name
+  api_url          = "https://api.github.com"
+  http_url         = "https://github.com"
+  oauth_token      = "my-vcs-provider-token"
+  service_provider = "github"
+}
+
+resource "tfe_registry_module" "test-registry-module" {
+  test_config {
+    tests_enabled = true
+  }
+
+  vcs_repo {
+    display_identifier = "my-org-name/terraform-provider-name"
+    identifier         = "my-org-name/terraform-provider-name"
+    oauth_token_id     = tfe_oauth_client.test-oauth-client.oauth_token_id
```

Comment:
wouldnt we need to add branch here as well?

Reply:
Updated